src/QuNLDE.jl

```diff
@@ -0,0 +1,86 @@
+export func_transform, euler_matrix, nonlinear_transform, make_input_vector, make_unitary
+
+function make_input_vector(x::Vector)
```

Comment:
you don't need to write in this way. You could just use a type parameter in the function head


```suggestion
function make_input_vector(x::Vector{T}) where T
```